dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -115,15 +115,15 @@ def update_channel_abuse_score(channel_id)
     # and signed in users to report.
     restrict_reporting_to_verified_users = DCDO.get('restrict-abuse-reporting-to-verified', true)
     amount =
-      if current_user&.verified_teacher?
+      if current_user&.verified_teacher? || current_user&.project_validator?
         20
       elsif current_user && !restrict_reporting_to_verified_users
         10
       else
         0
       end
     begin
-      value = Projects.new(get_storage_id).increment_abuse(channel_id, amount)
+      value = Projects.new(get_storage_id).increment_abuse(channel_id, amount, current_user&.project_validator?)
```

Comment:
nit: mind adding a comment here that project validators can change the abuse score on frozen projects?